main.py

```diff
@@ -35,13 +32,17 @@ def init_logging(default_level=logging.INFO):
         logging.basicConfig(level=default_level)
     logging.info('Successfully initialized logger.')
 
+#move everything down so init_loggin is defined
+init_logging()#start logging bfore creating the application so that __init__.py log will stay in logging_dev/loggin_prod
```

Comment:
Move the `# Start logging` comment to the line above `init_logging()`
Also there is a typo in the comment. And maybe word as `... __init__.py will log to logging_dev/logging_prod.`

Reply:
What do you mean by word as?